package.json

```diff
@@ -52,14 +53,14 @@
     "eslint-plugin-jsx-a11y": "^2.2.3",
     "eslint-plugin-react": "^6.1.2",
     "istanbul": "1.0.0-alpha.2",
+    "json-loader": "^0.5.4",
     "nsp": "^2.6.1",
     "tap-spec": "^4.1.1",
     "tape": "^4.5.1"
   },
   "dependencies": {
-    "roi": "^0.14.0",
```

Comment:
I haven't been following as closely as I probably should have, but what's the reason for replacing roi with fetch?